src/sui/themes/rtd-site/elements/icon.overrides

```diff
@@ -0,0 +1,2062 @@
+/*******************************
+           Variations
+*******************************/
+
+/*-------------------
+       Colors
+--------------------*/
+
+/* Red */
+i.light.icon {
+  color: @lightGrey !important;
+}
+i.inverted.light.icon {
+  color: @grey !important;
+}
+i.inverted.bordered.light.icon,
+i.inverted.circular.light.icon {
+  background-color: @lightGrey !important;
+  color: @black !important;
+}
+
+
+/*******************************
```

Comment:
👍 I like this idea. Although we might want to use the same pattern here as the one we'll use on including our own components. If that is so then this can be dependant on the decision there also.

Reply:
I would say we can get away with a much lighter version of what you were trying to accomplish. For instance, I'm describing a `icon.overrides` similar to:

```less
@import "icons/fontawesome.overrides";

i.inverted.light.icon {
  color: @grey !important;
}
```

or `icons.fontawesome.overrides` or something. I could see us using a similar pattern if any of the overrides get to be unmanageably large (however SUI overrides are already so fragmented that might be hard). The thing that I think we'll get confused by is separating components into per-template components.

Reply:
> The thing that I think we'll get confused by is separating components into per-template components.

Ok. This clears up several other discussions on dropping the templates/.overrides files. I think I get where you are going with this line of thought now.

As for the current conversation:
* I created 2 new files `icon.font.overrides` and  `icon.font.overrides`.
* I left the font name - _fontawesome_ - out of the file name so we are free to change the file contents and origin without changing the file itself.
* Moved the font generated content to the `.icon.font` files
* Left our custom content in the original `.icon` files